registry/storage/driver/s3-aws/s3.go

```diff
@@ -287,6 +291,14 @@ func FromParameters(parameters map[string]interface{}) (*Driver, error) {
 		return nil, fmt.Errorf("the skipVerify parameter should be a boolean")
 	}
 
+	caBundle, present := os.LookupEnv("AWS_CA_BUNDLE")
+	if present {
+		// unset AWS_CA_BUNDLE so AWS session won't raise LoadCustomCABundleError
+		os.Unsetenv("AWS_CA_BUNDLE")
```

Comment:
Clearly the AWS SDK knows about the `AWS_CA_BUNDLE` environment variable. Why can't we defer to the AWS SDK to handle checking the environment and loading the CA certificate bundle itself?

Reply:
I would also be in favour of less (or potentially no) code changes if the SDK can pick up the bundle automagically.